src/code.cloudfoundry.org/cf-tcp-router/configurer/haproxy/config_marshaller_test.go

```diff
@@ -350,4 +358,155 @@ backend backend_80
 			})
 		})
 	})
+
+	Describe("Marshal to appropriate HAProxy stanza", func() {
+		Context("Given a TCP Route Mapping", func() {
+			It("for a simple route", func() {
```

Comment:
This is not descriptive. What are you actually trying to test here that isn't tested elsewhere?